rerun_cpp/src/rerun/blueprint/space_view_component.cpp

```diff
@@ -0,0 +1,76 @@
+// DO NOT EDIT! This file was auto-generated by crates/re_types_builder/src/codegen/cpp/mod.rs
+// Based on "crates/re_types/definitions/rerun/blueprint/space_view.fbs".
+
+#include "space_view_component.hpp"
+
+#include <arrow/builder.h>
+#include <arrow/type_fwd.h>
+
+namespace rerun {
+    namespace blueprint {
+        const std::shared_ptr<arrow::DataType> &SpaceViewComponent::arrow_datatype() {
+            static const auto datatype = arrow::struct_({
+                arrow::field(
+                    "space_view",
+                    arrow::list(arrow::field("item", arrow::uint8(), false)),
+                    false
+                ),
+            });
+            return datatype;
+        }
+
+        Result<std::shared_ptr<arrow::StructBuilder>> SpaceViewComponent::new_arrow_array_builder(
+            arrow::MemoryPool *memory_pool
+        ) {
+            if (!memory_pool) {
```

Comment:
You are still on the old `.clang-format` file. I recently added `DerivePointerAlignment: false` to it that ensured pointer-stars get grouped with the type, so `arrow::MemoryPool* memory_pool`.

You should merge in `main` and run `just codegen` again before merging